src/SDKs/Azure.ApplicationModel.Configuration/data-plane/Azure.ApplicationModel.Configuration/ConfigurationClient_private.cs

```diff
@@ -138,56 +116,54 @@ Uri BuildUriForLocksRoute(string key, string label)
             return builder.Uri;
         }
 
-        void BuildBatchQuery(UriBuilder builder, BatchRequestOptions options)
+        void BuildBatchQuery(UriBuilder builder, ConfigurationSelector selector)
```

Comment:
This routine builds URIs from user input. Is it safe? i.e. does UriBuilder escape everything correctly?

cc: @GrabYourPitchforks?

Reply:
I don't know what `UriBuilder.AppendQuery` is. Perhaps a private extension method on the public `UriBuilder` type? I can't find the implementation of it right now.